tests/test_chats_list_page.py

```
@@ -0,0 +1,291 @@
+        status="active",
+        anchor_kind="standalone",
+        anchor_ref="",
+        anchor_title="",
+        profile="balanced",
+        last_message_at="",
+        turn_count=0,
+    )
+    try:
+        row.status = "pinned"  # type: ignore[misc]
+        raised = False
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)